package.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Please switch to pnpm instead of using npm, it should help with the performance

Reply:
Done. It will be included in the next branch commit.